container/lxd/network.go

```diff
@@ -327,10 +336,14 @@ const BridgeConfigFile = "/etc/default/lxd-bridge"
 // installations that pre-date the network API support and that were installed
 // via Snap. The question of the correct user action was posed on the #lxd IRC
 // channel, but has not be answered to-date.
-func checkBridgeConfigFile(reader func(string) ([]byte, error)) (string, error) {
-	bridgeConfig, err := reader(BridgeConfigFile)
+func checkBridgeConfigFile(fileName string, reader func(string) ([]byte, error)) (string, error) {
```

Comment:
I wonder if we should just keep the old signature and choose the _file-path_ based on whether we are Snap installed rather that trying both possibilities, and if we fail choosing an _error message_ based on Snap detection.

Reply:
yeah, probably, i'll change that